### PR TITLE
Update managing-carvel-packages tutorial with instructions for new repo UI

### PR DESCRIPTION
**Pull request by ppbaena**
### Description of the change

Update managing-carvel-packages tutorial including the instructions to install Package Repositories directly for new repo UI (replacing the manual configuration of Package Repositories)

### Benefits

Updated documentation and simplified path to configure Package repositories for Carvel.

### Possible drawbacks

Losing the instructions to manually configure a Package Repository in Carvel.

### Applicable issues

- related #5220 

### Additional information
Minor typo in the getting started guide included  to comply with style guidelines .

### CODE REVIEW

File: site/content/docs/latest/tutorials/managing-carvel-packages.md [hunk 1]
```diff
@@ -34,19 +32,19 @@ This guide walks you through the process of using Kubeapps for configuring and d
 
 > **NOTE**: This section can safely be skipped if you already have kapp-controller installed in your cluster.
 
-In order to manage Carvel Packages, first of all, you will need to install kapp-controller in your cluster. That is, applying a set of Kubernetes resources and CRDs.
+In order to manage Carvel Packages, first of all, you need to install kapp-controller in your cluster. That is, applying a set of Kubernetes resources and CRDs.
 
 According to the [Carvel kapp-controller official documentation](https://carvel.dev/kapp-controller/docs/latest/install/), you can install everything it kapp-controller requires just by running the following command:
```
Comment (castelblanque) [inline site/content/docs/latest/tutorials/managing-carvel-packages.md:37]:
```suggestion
According to the [Carvel kapp-controller official documentation](https://carvel.dev/kapp-controller/docs/latest/install/), you can install everything that kapp-controller requires just by running the following command:
```

File: site/content/docs/latest/tutorials/managing-carvel-packages.md [hunk 2]
```diff
@@ -201,46 +198,46 @@ The reason why is that kapp-controller forces to explicitly provide needed privi
...
+As in any other packaging format, this page displays those Kubernetes resources that have been created as a result of the Package installation.
 Besides, the current values are shown at the end of the page.
 
-Next, you can click on the `Delete` button to uninstall the application or the `Upgrade` button to edit the values of the application or update it to another version.
+Next, you can click on the **Delete** button to uninstall the application or the **Upgrade** button to edit the values of the application or update it to another version.
 
-> **NOTE**: as opposed to Helm Charts, Carvel Packages cannot be rolled back, hence there is no `Rollback` button.
+> **NOTE**: as opposed to Helm Charts managed by using Helm, Carvel Packages cannot be rolled back, hence there is no **Rollback** button.
 
 Finally, note that every Carvel Package installed through Kubeapps can also be managed by the [kapp](https://carvel.dev/kapp/) CLI using the `kapp inspect -a <APPLICATION_NAME>` command. For example:
```
Comment (castelblanque) [inline site/content/docs/latest/tutorials/managing-carvel-packages.md:242]: Here we are saying that stuff installed in Kubeapps is shown with kapp CLI.
Shouldn't we also mention the other way around? i.e. that packages installed from kapp CLI are also presented in Kubeapps.

**Review by castelblanque — APPROVED** [review]
Awesome work, many thanks!

File: site/content/docs/latest/tutorials/managing-carvel-packages.md (hunk 2 above)
Comment (ppbaena) [reply site/content/docs/latest/tutorials/managing-carvel-packages.md:242]: Agree

Comment (ppbaena) [inline site/content/docs/latest/tutorials/managing-carvel-packages.md:242]:
```suggestion
Finally, note that every Carvel Package installed through Kubeapps can also be managed by the [kapp](https://carvel.dev/kapp/) CLI using the `kapp inspect -a <APPLICATION_NAME>` command (as well as every Carvel Package installed directly from the kapp CLI can also be managed by Kubeapps). For example:
```

**Pull request merged**